velox/functions/sparksql/MightContain.cpp

```diff
@@ -33,26 +49,24 @@ class BloomFilterMightContainFunction final : public exec::VectorFunction {
     context.ensureWritable(rows, BOOLEAN(), resultRef);
     auto& result = *resultRef->as<FlatVector<bool>>();
     exec::DecodedArgs decodedArgs(rows, args, context);
-    auto serialized = decodedArgs.at(0);
     auto value = decodedArgs.at(1);
 
-    HashStringAllocator allocator{context.pool()};
-    VELOX_USER_CHECK(serialized->isConstantMapping())
-    BloomFilter output{StlAllocator<uint64_t>(&allocator)};
-    try {
-      auto sv = serialized->valueAt<StringView>(0);
-      output.merge(sv.data());
-    } catch (const std::exception& e) {
-      context.setErrors(rows, std::current_exception());
-      return;
+    if (!isSet_) {
+      rows.applyToSelected([&](int row) { result.set(row, false); });
+    } else {
+      rows.applyToSelected([&](int row) {
```

Comment:
Can we initialize the bloomFilter_ here? Then we can use context pool to track memory usage

Reply:
No, because this function is marked with `const`.

Reply:
the function being const should not be a limitation you can always mark bloomFilter_ as mutable